### PR TITLE
Imageapi optimize

### PR DESCRIPTION
**Pull request by MuschPusch**
Adding support for image_optimize pipelines. Will send you another PR using it in one our projects.

### CODE REVIEW

**Review by d34dman — COMMENTED** [review]
Looks good and non-breaking 👍 
Could we have test please?

**Comment by d34dman** [comment]
Ah sorry, forgot that we don't need to write php code for tests :P, ignore my previous comment

File: src/transformers/ImageStyleTransformer.php [hunk 1]
```diff
@@ -76,6 +77,11 @@ public function transform(
                         $result['effects'][PlaceholderService::createChildReference('uuid')] = $result_effect;
                     }
 
+                    // Image optimise pipeline support.
+                    if (isset($style['pipeline'])) {
+                        // $result['dependencies']['module'][] = 'focal_point';
```
Comment (d34dman) [inline src/transformers/ImageStyleTransformer.php:82]: Un-necessary comment?

**Review by d34dman — COMMENTED** [review]
Just a tiny comment... i think this one is good to go in!

File: src/transformers/ImageStyleTransformer.php (hunk 1 above)
Comment (MuschPusch) [reply src/transformers/ImageStyleTransformer.php]: Removed it!

**Pull request merged**